praktikumsplaner-frontend/frontend/src/views/nachwuchskraefte/NachwuchskraefteView.vue

```diff
@@ -22,16 +22,15 @@
 </template>
 
 <script setup lang="ts">
-import { ref } from "vue";
 import ExcelImportNwk from "@/components/nachwuchskraefte/ExcelImportNwk.vue";
 import PageTitle from "@/components/common/PageTitle.vue";
 import ActiveNwkList from "@/components/nachwuchskraefte/ActiveNwkList.vue";
 import NwkCreateDialog from "@/components/nachwuchskraefte/NwkCreateDialog.vue";
+import { EventBus } from "@/stores/event-bus";
 
-const nwkList = ref();
-function created() {
-    nwkList.value.loadAllActiveNwks();
-}
+EventBus.$on("nwkCreated", () => {
+    EventBus.$emit("nwkCreated");
+});
```

Comment:
That serves no purpose as the eventbus is global. So basically you are emitting the same event twice